http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/ZombieMonitorQueries.scala

```diff
@@ -0,0 +1,32 @@
+package org.broadinstitute.dsde.workbench.leonardo.db
+
+import cats.data.Chain
+import cats.implicits._
+import org.broadinstitute.dsde.workbench.leonardo.AuditInfo
+import org.broadinstitute.dsde.workbench.leonardo.db.LeoProfile.api._
+import org.broadinstitute.dsde.workbench.leonardo.model.google.{ClusterName, ClusterStatus}
+import org.broadinstitute.dsde.workbench.model.google.GoogleProject
+import slick.dbio.DBIO
+
+import scala.concurrent.ExecutionContext
+
+object ZombieMonitorQueries {
+
+  def listZombieQuery(implicit ec: ExecutionContext): DBIO[Map[GoogleProject, Chain[PotentialZombieCluster]]] =
+    clusterQuery.filter { _.status inSetBind ClusterStatus.activeStatuses.map(_.toString) }.result.map { cs =>
+      cs.toList.foldMap { c =>
+        Map(
+          c.googleProject -> Chain(
```

Comment:
Huh, this is a cool data structure

Reply:
yeah I learned about it a little while ago, it's a like a more efficient `List`

https://typelevel.org/cats/datatypes/chain.html